libs/ui-lib/lib/ocm/components/clusters/ClusterStatus.tsx

```diff
@@ -26,22 +27,46 @@ type ClusterStatusIconProps = {
 export const ClusterStatusIcon: React.FC<ClusterStatusIconProps> = ({ status, ...extraProps }) => {
   switch (status) {
     case 'cancelled':
-      return <BanIcon size={'sm'} {...extraProps} />;
+      return (
+        <Icon size="sm">
+          <BanIcon {...extraProps} />
+        </Icon>
+      );
     case 'insufficient':
     case 'pending-for-input':
     case 'ready':
-      return <FileAltIcon size={'sm'} {...extraProps} />;
+      return (
+        <Icon size="sm">
+          <FileAltIcon {...extraProps} />
+        </Icon>
+      );
     case 'error':
-      return <ExclamationCircleIcon color={dangerColor.value} size={'sm'} {...extraProps} />;
+      return (
+        <Icon size="sm">
+          <ExclamationCircleIcon color={dangerColor.value} {...extraProps} />
+        </Icon>
+      );
     case 'installed':
-      return <CheckCircleIcon color={okColor.value} size={'sm'} {...extraProps} />;
+      return (
+        <Icon size="sm">
+          <CheckCircleIcon color={okColor.value} {...extraProps} />
+        </Icon>
+      );
     case 'installing-pending-user-action':
-      return <ExclamationTriangleIcon color={warningColor.value} size={'sm'} {...extraProps} />;
+      return (
+        <Icon size="sm">
```

Comment:
I saw that in other places where we converted the icons to be wrapped in `<Icon>`, we removed the color and used a `variant` instead. Maybe be best to do it if possible.